cvprac/cvp_api.py

```diff
@@ -3790,6 +3790,46 @@ def device_decommissioning_status_get_all(self, status="DECOMMISSIONING_STATUS_U
             self.log.debug('v7 ' + str(url))
             return self.clnt.post(url, data=payload, timeout=self.request_timeout)
 
+    def add_role(self, name, description, moduleList):
+        ''' Add new local role to the CVP UI.
+            Args:
+                name (str): local role name on CVP
+                description (str): role description
+                moduleList (str): list of modules (name (str) and mode (str))
```

Comment:
Is this a list object or a string with some specific list formatting?

Reply:
It's a list. Updated.